packages/node_modules/@webex/internal-plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -279,5 +323,112 @@ describe.skip('plugin-calendar', () => {
           });
       });
     });
+    describe('#meetingNotes', () => {
+      let creator, mccoy, webex, spock;
+
+      before('create test users', () => testUsers.create({
+        count: 3,
+        config: {
+          entitlements: [
+            'webex',
+            'squaredCallInitiation',
+            'squaredInviter',
+            'squaredRoomModeration',
+            'webExSquared',
+            'squaredFusionCal'
+          ]
+        }
+      })
+        .then((users) => {
+          console.log('Enter then');
+          [creator, spock, mccoy] = users;
+          console.log('Got creator, spock, and mccoy');
+
+          webex = new WebexCore({
+            credentials: {
+              authorization: creator.token
+            },
+            config: {
+              services: {
+                discovery: {
+                  whistlerServiceUrl: process.env.WHISTLER_API_SERVICE_URL || 'https://calendar-whistler.onint.ciscospark.com:8084/api/v1'
+                }
+              }
+            }
+          });
+          console.log('Got webexcore');
+        }));
+
+      before('register to wdm, set features, and connect to mercury', () => webex.internal.device.register()
+        .then(() => webex.internal.feature.setFeature('developer', 'calsvc_calendar_view', true))
+        .then(() => webex.internal.mercury.connect()));
+
+      after(() => webex && webex.internal.mercury.disconnect());
+
+      it('retrieves the meeting notes for the given meetingId', () => {
+        const meetingNotes = 'Dummy meeting notes';
+        const meetingID = uuid.v4();
+        const locusID = uuid.v4();
+        const seriesID = uuid.v4();
+
+        const meetingParams = {
+          meetingId: meetingID,
+          seriesId: seriesID,
+          locusId: locusID,
+          meetingNotes
+        };
+
+        return postNotesToWhistler(webex, 'CREATE', makeMockMeetingPayload(meetingParams))
+          .then((res) => {
```

Comment:
If you follow my suggestion above, this could be:
```suggestion
          .then((createMeeting) => {
```
and you can delete the next line